packages/api/src/routes/internal/hie/commonwell.ts

```diff
@@ -107,4 +111,43 @@ router.put(
   })
 );
 
+/**
+ * POST /internal/commonwell/patient-discovery/:patientId
+ *
+ * Triggers patient discovery for a specific patient in CommonWell.
+ * @param req.query.cxId The customer ID.
+ * @param req.params.patientId The ID of the patient to run discovery for
+ * @param req.query.rerunPdOnNewDemographics Optional flag to rerun discovery if new demographics are found
+ * @param req.query.forceCommonwell Optional flag to force CommonWell discovery
+ * @returns 200 OK if discovery was triggered successfully
+ */
+router.post(
+  "/patient-discovery/:patientId",
```

Comment:
We already have an internal PD route, `POST /internal/patient/:id/patient-discovery`, can't we just add a param there to limit to a specific HIE?

Reply:
Curious about this as well, as normally :patientId would come immediately after /patient/ in the routes, so something like

`POST /internal/patient/:id/patient-discovery?myFlag=abcd`

would be better

Reply:
I didnt want to update the function that endpoint calls `runOrSchedulePatientDiscoveryAcrossHies`. This would be a bigger change so i figured it would be better to scope to a new endpoint.

Reply:
And directly call runOrScheduleCwPatientDiscovery instead

Reply:
I think more params is sometimes worse, so fine by me. In that case we could make the route this:

`POST /internal/patient/:id/patient-discovery/commonwell`

Reply:
Im using the same paradigm as the endpoints above /internal/commonwell/ops/facility/:oid given this is specific to CW

Reply:
I could move to patient file instead but i think it could go either way